### PR TITLE
pin to a minimum winrm-s of 0.3.2

### PR DESCRIPTION
**Pull request by mwrock**

### CODE REVIEW

**Comment by jaym** [comment]
:+1:

File: knife-windows.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version	= ">= 1.9.1"
   s.add_dependency "winrm", "~> 1.3"
-  s.add_dependency "winrm-s", "~> 0.3.0"
+  s.add_dependency "winrm-s", "~> 0.3.0", ">= 0.3.2"
```
Comment (btm) [inline knife-windows.gemspec]: ` s.add_dependency "winrm-s", "~> 0.3.2"` ?

**Comment by btm** [comment]
:+1: to get people https://github.com/chef/winrm-s/pull/29, but I think you can get by with one constraint there.

File: knife-windows.gemspec (hunk 1 above)
Comment (mwrock) [inline knife-windows.gemspec]: right. i was thinking of the ~> 0.3 being the first constraint which clearly it is not. will change to what you have above

**Pull request merged**